package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "contain-facebook",
+  "version": "1.2.4",
+  "description": "Facebook Container isolates your Facebook activity from the rest of your web activity in order to prevent Facebook from tracking you outside of the Facebook website via third party cookies. ",
+  "main": "background.js",
+  "scripts": {
+    "test": "eslint .",
+    "build": "npm test && web-ext build --overwrite-dest"
```

Comment:
This _should_ skip the package.json and package-lock.json files when building the XPI:

```js
"build": "npm test && web-ext build --overwrite-dest --ignore-files 'package*'"
```

Results:

```sh
92K  facebook_container-1.2.4.zip # with package.json and package-lock.json
26K  facebook_container-1.2.5.zip # with `--ignore-files 'package*'`
```